src/Lykke.Logs/Obsolete/LykkeLogToAzureSlackNotificationsManager.cs

```diff
@@ -4,20 +4,24 @@
 using System.Threading.Tasks;
 using Common;
 using Common.Log;
+using JetBrains.Annotations;
+using Lykke.Common;
 using Lykke.SlackNotifications;
 
 namespace Lykke.Logs
 {
     /// <summary>
     /// Class for async sending of slack messages from internal queue.
     /// </summary>
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
See above

Reply:
There is no direct replacement for this class